### PR TITLE
Email is sent when new post in group

### PR DESCRIPTION
**Pull request by janimattiellonen**

### CODE REVIEW

File: src/Cobase/AppBundle/Controller/GroupController.php [hunk 1]
```diff
@@ -419,6 +419,65 @@ public function feedAction($groupId)
...
+            return $this->render('CobaseAppBundle:Group:notfound.html.twig',
+                $this->mergeVariables()
+            );
+        }
+
+        if ($notificationService->isUserUserNotifiedOfNewGroupPosts($group, $user)) {
+            $this->get('session')->getFlashBag()->add(
+                'group.message',
+                'You are already notified when a new post is made to the group "' . $group->getTitle() . '"');
+            return $this->redirect($this->generateUrl('CobaseAppBundle_group_view', ['groupId' => $groupId]));
```
Comment (janimattiellonen) [inline src/Cobase/AppBundle/Controller/GroupController.php]: If we allow php 5.3 then I need to change these [ to array()

File: src/Cobase/AppBundle/Entity/Notification.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass="Cobase\AppBundle\Repository\NotificationRepository")
+ * @ORM\Table(name="notifications")
+ */
+class Notification
+{
+    use TimestampTrait;
```
Comment (janimattiellonen) [inline src/Cobase/AppBundle/Entity/Notification.php]: Cannot use if we plan to support php 5.3

**Comment by artur-gajewski** [comment]
mailer_from in parameters.yml-dist should be admin@example.com, not example.com as this values gives an exception.

**Comment by artur-gajewski** [comment]
Also, add one space before the link for the Notify/Unnotify link's text to seperate the text from the envelope icon.

**Pull request merged**